src/CosmosDB/CosmosDB/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 
 ## Upcoming Release
+## Version 1.0.0
```

Comment:
The headings for versions are added by script and should not be udpated manually
```suggestion
```